graylog2-web-interface/src/views/components/SearchResult.tsx

```diff
@@ -56,6 +67,7 @@ type Props = {
 
 const SearchResult = React.memo(({ queryId, searches, viewState }: Props) => {
   const fieldTypes = useContext(FieldTypesContext);
+  const focusedWidget = useContext(WidgetFocusContext);
```

Comment:
We need to use ` const { focusedWidget } = useContext(WidgetFocusContext);` here as well, right now `focusedWidget`  is always true.